src/CabanaPD_Boundary.hpp

```diff
@@ -165,12 +178,18 @@ struct BoundaryCondition
                 auto pid = index_space( b );
                 user( pid );
             } );
+        _timer.stop();
     }
+
+    auto time() { return _timer.time(); };
+    auto timeInit() { return _index_space.time(); };
```

Comment:
What does this line do compared to:

 auto timeInit() { return 0.0; };

Reply:
This returns the result from line 124, not zero